src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -8092,4 +8092,28 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ExtensionResolutionFailed" xml:space="preserve">
     <value>'{0}' does not contain a definition for '{1}' and no accessible extension member '{1}' for receiver of type '{0}' could be found (are you missing a using directive or an assembly reference?)</value>
   </data>
+  <data name="ERR_ReceiverParameterSameNameAsTypeParameter" xml:space="preserve">
+    <value>'{0}': a receiver parameter cannot have the same name as an extension container type parameter</value>
+  </data>
+  <data name="ERR_LocalSameNameAsExtensionTypeParameter" xml:space="preserve">
+    <value>'{0}': a parameter, local variable, or local function cannot have the same name as an extension container type parameter</value>
+  </data>
+  <data name="ERR_TypeParameterSameNameAsExtensionTypeParameter" xml:space="preserve">
+    <value>Type parameter '{0}' has the same name as an extension container type parameter</value>
+  </data>
+  <data name="ERR_LocalSameNameAsExtensionParameter" xml:space="preserve">
+    <value>'{0}': a parameter, local variable, or local function cannot have the same name as an extension parameter</value>
```

Comment:
>an extension parameter [](http://example.com/codeflow?start=94&length=22)

Why "*an* extension parameter" as opposed to "*the* extension parameter"?
Also applies to other messages

Reply:
I used the following message as a template "'{0}': a parameter, local variable, or local function cannot have the same name as a method type parameter"